tests/50-sd-verify.js

```diff
@@ -2,291 +2,175 @@
-      testVectors.disclosed.noIds = await deriveTestData({
-        selectivePointers: subjectNestedObjects.selectivePointers.slice(1, 3),
-        verifiableCredential: signedDlCredentialNoIds,
-        keyTypes,
-        suite
+    it('MUST verify a valid VC with nested disclosed properties.',
+      async function() {
+        assertTestVector(nestedDisclosedCredentials);
+        for(const credential of nestedDisclosedCredentials) {
+          await verificationSuccess({credential, verifier});
+        }
       });
-      const credentialHasArrays = klona(subjectHasArrays);
-      // start third round test data creation w/
-      // AchievementCredential
-      const achievementCredentials = await issueTestData({
-        credential: credentialHasArrays.document,
-        mandatoryPointers: credentialHasArrays.mandatoryPointers,
-        keyTypes,
-        suite
+    it('MUST verify a valid VC with disclosed properties and bnodes.',
+      async function() {
+        assertTestVector(disclosedDlCredentialNoIds);
+        for(const credential of disclosedDlCredentialNoIds) {
+          await verificationSuccess({credential, verifier});
+        }
       });
-      const signedAchievementCredential = achievementCredentials.get(
-        keyTypes[0]);
-      // select full arrays
-      testVectors.disclosed.array.full = await deriveTestData({
-        selectivePointers:
-          [...credentialHasArrays.selectivePointers],
-        verifiableCredential: signedAchievementCredential,
-        suite,
-        keyTypes
+    it('MUST verify with full array revealed properties',
+      async function() {
+        assertTestVector(disclosedCredentialsWithFullArray);
+        for(const credential of disclosedCredentialsWithFullArray) {
+          await verificationSuccess({credential, verifier});
+        }
       });
-      // select less than full subarrays
-      const lessThanFullPointers = credentialHasArrays.
-        selectivePointers.slice(2, -4);
-      testVectors.disclosed.array.lessThanFull = await deriveTestData({
-        selectivePointers: lessThanFullPointers,
-        verifiableCredential: signedAchievementCredential,
-        suite,
-        keyTypes
+    it('MUST verify with fewer array revealed properties',
+      async function() {
+        assertTestVector(disclosedCredentialsWithLessThanFullSubArray);
+        for(const credential of
+          disclosedCredentialsWithLessThanFullSubArray) {
+          await verificationSuccess({credential, verifier});
+        }
       });
-      // select w/o first 7 array element
-      const removeFirst7Pointers = credentialHasArrays.
-        selectivePointers.slice(7);
-      testVectors.disclosed.array.missingElements = await deriveTestData({
-        selectivePointers: removeFirst7Pointers,
-        verifiableCredential: signedAchievementCredential,
-        suite,
-        keyTypes
+    it('MUST verify w/o first element revealed properties',
```

Comment:
can this fit on a single line now? It seems like this is less indented than previously.